payments/card/other-features.md

```diff
@@ -688,6 +688,61 @@ credit card payment:
   decided by the issuing bank. Normally this will be done using BankID or Mobile
   BankID.
 
+```mermaid
+sequenceDiagram
+    participant Payer
+    participant Merchant
+    participant SwedbankPay as Swedbank Pay
+    participant IssuingBank
+
+  activate Payer
+  Payer->>+Merchant: start verification
+  deactivate Payer
+  Merchant->>+SwedbankPay: POST /psp/creditcard/payments(operation=VERIFY)
+  deactivate Merchant
+  note left of Payer: First API request
+  SwedbankPay-->+Merchant: payment resource
+  deactivate SwedbankPay
+  Merchant-->>+Payer: redirect to verification page
+  deactivate Merchant
+  Payer->>+SwedbankPay: access verification page
+  deactivate Payer
+  note left of Payer: redirect to SwedbankPay<br>(If Redirect scenario)
+  SwedbankPay-->>+Payer: display purchase information
+  deactivate SwedbankPay
+
+  Payer->>Payer: input creditcard information
+  Payer->>+SwedbankPay: submit creditcard information
+  deactivate Payer
+  opt Card supports 3-D Secure
+    SwedbankPay-->>Payer: redirect to IssuingBank
+    deactivate SwedbankPay
+    Payer->>IssuingBank: 3-D Secure authentication process
+    Payer->>+SwedbankPay: access authentication page
+    deactivate Payer
+  end
+
+  SwedbankPay-->>+Payer: redirect to merchant
+  deactivate SwedbankPay
+  note left of Payer: redirect back to merchant<br>(If Redirect scenario)
+
+  Payer->>+Merchant: access merchant page
+  Merchant->>+SwedbankPay: GET /psp/creditcard/payments/<paymentorder.id>
+  deactivate Merchant
+  note left of Merchant: Second API request
+  SwedbankPay-->>+Merchant: rel: redirect-authorization
+  deactivate SwedbankPay
+  Merchant-->>Payer: display purchase result
+  deactivate Merchant
+
+  opt Callback is set
+    activate SwedbankPay
+    SwedbankPay->>SwedbankPay: Payment is updated
+    SwedbankPay->>Merchant: POST Payment Callback
+    deactivate SwedbankPay
+  end
+```
+
 ## Payment Orders
```

Comment:
Payment Orders are the technical underpinning of Checkout, not Payments, and should not be mentioned once in the Card Payments section. Please remove.

Reply:
this is now done :D